### PR TITLE
Frontend: add watermark to challenge leaderboards

### PR DESCRIPTION
**Pull request by matthew-so**
Why:
Challenge leaderboards can currently be screenshotted and stolen by visitors. Adding a watermark to the challenge leaderboard will discourage this and improve traffic to the website.

How:
The frontend has been updated to use the EvalAI logo as a watermark on the table. Examples can be seen below.

No leaderboard
Single result leaderboard
Moderately-sized leaderboard
Large leaderboard

### CODE REVIEW

File: frontend/src/css/modules/challenge.scss [hunk 1]
```diff
@@ -284,3 +284,11 @@ md-select .md-select-value span:first-child:after {
 .margin-bottom-cancel {
     margin-bottom: 0px;
 }
+
+.watermarked {
+    background-image: linear-gradient(rgba(255,255,255,0.9), rgba(255,255,255,0.9)),url(../images/evalai-logo-centered.png);
```
Comment (Ram81) [inline frontend/src/css/modules/challenge.scss:289]: @matthew-so are you sure the logo path will work with the deployment as well? I think the path needs to be `dist/images/evalai-logo.png`. Can you please check other image URL usages and fix this?

**Review by Ram81 — CHANGES_REQUESTED** [review]
Minor change needed. Rest of the PR LGTM :+1: 

Nicely done @matthew-so !

File: frontend/src/css/modules/challenge.scss (hunk 1 above)
Comment (matthew-so) [reply frontend/src/css/modules/challenge.scss:289]: I am sure; relative URLs are regularly used in the frontend code, as can be seen here: relative_path_example_evalai

**Review by Ram81 — CHANGES_REQUESTED** [review]
@matthew-so can you add the check to show watermark only on staging and dev environments?

**Comment by matthew-so** [comment]
I've added a check so that this watermark only appears while in staging or dev mode.

**Comment by matthew-so** [comment]
Rishabh recommended that the watermark be rotated for security. I've completed this change and attached some screenshots of the change. Let me know if changes should be made at this point.
Diagonal_Watermark_Small_Screen_Single_Row
Diagonal_Watermark_Full_Screen_Single_Row
Diagonal_Watermark_Small_Window
Diagonal_Watermark_Multiple_Rows

**Comment by matthew-so** [comment]
In the interests of aesthetics, the background watermark has been reverted to a single horizontal one.
Screenshot 2022-10-23 015051

**Comment by matthew-so** [comment]
The watermark has been reformed into a single horizontal watermark (in order to maintain the table's aesthetics). It is now anchored to the top of the table, since this is where most screenshots occur. Should we make this watermark repeat vertically as well?
Screenshot 2022-10-23 020004

**Review by Ram81 — CHANGES_REQUESTED** [review]
@matthew-so can you resolve the merge conflicts?

**Comment by matthew-so** [comment]
Just merged the change

**Review by Ram81 — APPROVED** [review]

**Pull request merged**